page-officers.hbs

```diff
@@ -0,0 +1,427 @@
+{{!--
```

Comment:
Would be nice if you could use a JSON format and then iterate through that to display some of the info, but it looks like Ghost is not capable of that. The manual HTML galore :(

Reply:
That would be nice, but I think leaving it hard-coded as is would be a good exercise for those who maintain the theme next.